game/definitions.rpy

```diff
@@ -281,6 +281,7 @@ init 0 python:
             affinity=None,
             trust=None,
             conditional=None,
```

Comment:
We should exclude `conditional` from this given we should always want to evaluate the conditional of a topic if it's present.